mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -91,80 +61,89 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
     return data_thresholded, center_locations
 
 
-def gridding_2d(points, values, img_shape, method='linear', point_min=None,
-                point_max=None):
+def grided_inverse_fourier_transform_nd(points, values, grid, method):
```

Comment:
I am sorry I had a typo in my suggested name, it's actually gridded.

Also I don't understand why you won't change the 'values' variable name to something like 'kspace_values' or 'fourier_coefficients'. To me it feels like 'values' is too broad and misleading (just like 'points' in some respect).

Reply:
Values came through from griddata, yes I had it changed in a commit which moved away from mainline :( 
Ill keep it is `kspace_data` and `kspace_loc`